.github/workflows/test.yml

```
@@ -88,6 +88,17 @@ jobs:
         source /opt/ros/rolling/setup.sh && colcon test --mixin linters-skip --packages-select ${rosbag2_packages} --packages-skip rosbag2_performance_benchmarking --event-handlers console_cohesion+ --return-code-on-test-failure --ctest-args "-L xfail" --pytest-args "-m xfail"
       working-directory: ${{ steps.action-ros-ci.outputs.ros-workspace-directory-name }}
       shell: bash
+    - name: Is regeneration of Python stubs required?
+      run: |
+        rosbag2_path=$(colcon list -p --packages-select rosbag2)/..
+        sudo pip uninstall -y mypy
+        sudo apt update && sudo apt -y install mypy
```

```suggestion
        sudo apt update && sudo apt -y install mypy=0.942-1ubuntu1
```

We should match `mypy` version to what we specified in the readme file.

---

@MichaelOrlov my bad, done